pkg/cosign/verify.go

```diff
@@ -1220,6 +1257,38 @@ func VerifyBundle(sig oci.Signature, co *CheckOpts) (bool, error) {
 	return true, nil
 }
 
+type signedEntityForTimestamp struct {
+	verify.BaseSignedEntity
+	timestamp  *cbundle.RFC3161Timestamp
+	sigContent *sigContent
+}
+
+type sigContent struct {
+	rawSig []byte
+}
+
+func (e *signedEntityForTimestamp) Timestamps() ([][]byte, error) {
+	timestamps := make([][]byte, 1)
+	timestamps[0] = e.timestamp.SignedRFC3161Timestamp
+	return timestamps, nil
+}
+
+func (e *signedEntityForTimestamp) SignatureContent() (verify.SignatureContent, error) {
+	return e.sigContent, nil
+}
+
+func (s *sigContent) Signature() []byte {
+	return s.rawSig
+}
+
+func (s *sigContent) EnvelopeContent() verify.EnvelopeContent {
+	panic("not implemented")
```

Comment:
Can we return nil? I'm hesitant to put any panics in here that might inadvertently get called.